frontend/src/pages/videos/VideoCard.tsx

```diff
@@ -18,12 +21,52 @@ const useStyles = makeStyles(() => ({
 function VideoCardPage() {
   const classes = useStyles();
   const { video_id } = useParams<{ video_id: string }>();
+  console.log(video_id);
+  const video = useVideoMetadata(video_id);
+  console.log(video);
+  const captions: any = video.criteria_scores
+    ? video.criteria_scores.length > 0
+      ? Object.fromEntries(
+          video.criteria_scores.map((criteria) => [
+            criteria.criteria,
+            criteria.criteria,
+          ])
+        )
+      : {
+          d: 'd',
+        }
+    : null;
 
+  const data: any = video.criteria_scores
+    ? video.criteria_scores.length > 0
+      ? {
+          data: Object.fromEntries(
+            video.criteria_scores.map((criteria) => [
+              criteria.criteria,
+              criteria.score,
+            ])
+          ),
+          meta: { color: 'purple' },
+        }
+      : {
+          data: {
+            d: 0,
+          },
+          meta: { color: 'blue' },
+        }
+    : null;
+  console.log(captions);
+  console.log(video.criteria_scores);
   return (
     <div>
       <div className={classes.root}>
         <VideoCardFromId videoId={video_id} />
       </div>
+      <Container maxWidth="sm">
+        {video.criteria_scores && (
```

Comment:
I suggest to not display the `RadarChart` when there is no score for now.

The current code displays an empty chart with a "placeholder" value which is a bit hacky. Let's not display the cart if we have no proper way to display it empty.

```suggestion
        {video.criteria_scores?.length > 0 && (
```